spotify-controller@koolskateguy89/metadata.json

```diff
@@ -4,7 +4,8 @@
   "shell-version": [
     "3.36",
     "3.38",
-    "40"
+    "40",
+    "42"
```

Comment:
The extension probably works fine with other versions but I'm not sure as I only have Gnome 40 installed. I will check what changed in Gnome 41 etc. and try and update the code for this and will update the supported version on ego soon.